aspnet/identity/overview/features-api/customize_identity_model.md

```diff
@@ -0,0 +1,807 @@
+---
+title: Identity model customization
+author: ajcvickers
+description: This article describes how to customize the underlying Entity Framework Core data model for ASP.NET Core Identity.
+manager: elipton
+ms.author: avickers
+ms.date: 04/12/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: 0BE376D3-FE2F-4A30-803A-74C2148FE5E4
```

Comment:
@Rick-Anderson Should this doc reside under the **Security** > **Authentication** ToC node? We'll need to adjust the UID accordingly.